packages/react-theme/src/themes/teams/darkTheme.ts

```diff
@@ -1,4 +1,5 @@
 import { createTeamsDarkTheme } from '../../utils/createTeamsDarkTheme';
 import { brandTeams } from '../../global/brandColors';
+import { Theme } from '../../types';
 
-export const teamsDarkTheme = createTeamsDarkTheme(brandTeams);
+export const teamsDarkTheme: Theme = createTeamsDarkTheme(brandTeams);
```

Comment:
Desperate fix for api-extractor.

Without explicitly typing the variable, typescript generates the following .d.ts:
```js
export declare const webLightTheme: import("../..").Theme;
```

which api-extractor fails on:

![image](https://user-images.githubusercontent.com/9615899/122912517-b445fb00-d358-11eb-8203-9e2b160cfabf.png)

Reply:
interesting ! what's the normal source import of that `Theme` ? we have `node ../../scripts/typescript/normalize-import` which normalizes this TS anomaly  (fixed in TS 4.3 )